### PR TITLE
Add ALC 230 layout 20 for Lenovo Ideapad 320 by maiconjs

### PR DESCRIPTION
**Pull request by maiconjs**
Please add new ALC230 corrected for Lenovo 320. Internal Speaker, Headphone, Mic Internal and Line in working. Mic Internal and Line In are separated in different stream.

### CODE REVIEW

**Comment by maiconjs** [comment]
layout40 for Asus B85-ME

**Comment by anmoolpk** [comment]
I hope this codec ALC230 layout id 20 will work my Lenovo ideapad 320
I m waiting this alc230 to add AppleALC Source

**Comment by paskmaksim** [comment]
Добавьте пожалуйста кодек ALC294 на ASUS A542UA

File: Resources/ALC230/Info.plist [hunk 1]
```diff
@@ -39,113 +55,149 @@
 			<key>Count</key>
 			<integer>1</integer>
 			<key>Find</key>
-			<data>QcYGAEmLvCQ=</data>
```
Comment (vit9696) [inline Resources/ALC230/Info.plist]: Please fix xml formatting here and below. AppleALC rebuild should do.

Comment (maiconjs) [reply Resources/ALC230/Info.plist]: Fixed ! Thanks 👍

**Comment by vit9696** [comment]
If you check it out, some files are out of order still. There are lots of non-functional changes on 
https://github.com/acidanthera/AppleALC/pull/315/files.

**Comment by vit9696** [comment]
Also, you added layout40 but did not describe it in Info.plist. Please ensure the changes are consistent.

**Comment by maiconjs** [comment]
Layout added. Many thanks again for the analysis in the files

**Comment by maiconjs** [comment]
I´ve tried all the layouts, and the behavior have is: the Mic and line-in do not work, Blue Line does not capture audio and SPDIF is not presented. These fixes cover it.

**Pull request merged**